keyboards/themadnoodle/udon13/info.json

```diff
@@ -0,0 +1,75 @@
+{
+    "manufacturer": "The Mad Noodle",
+    "keyboard_name": "Udon13",
+    "maintainer": "The-Mad-Noodle",
+    "url": "https://www.madnoodleprototypes.com/",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "encoder": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "oled": true
+    },
+    "rgblight": {
+        "hue_steps": 10,
+        "led_count": 6,
+        "sleep": true,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "B7"
+    },
+    "matrix_pins": {
+        "cols": ["F0", "F5", "F4", "F6"],
+        "rows": ["F7", "F1", "D6", "D7"]
+    },
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0012",
+        "vid": "0x6A6C"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x4": {
```

Comment:
This is not the `ortho_3x4` layout (it would be if you did not have the extra encoder):
![layout image](https://github.com/qmk/qmk_firmware/assets/616082/5165bd1c-9179-477d-ae18-cb27c6513626)
```suggestion
        "LAYOUT": {
```